src/foam/java/refinements.js

```diff
@@ -70,7 +70,8 @@ foam.CLASS({
         propRequired: this.required,
         jsonParser:   this.javaJSONParser,
         extends:      this.javaInfoType,
-        transient:    this.transient
+        transient:    this.transient,
```

Comment:
I think we need both networkTransient and storageTransient, but don't actually need 'transient'.

Reply:
I've added networkTransient and storageTransient. But why would I remove just transient?